angr/__init__.py

```diff
@@ -16,6 +16,9 @@
 Good luck!
 """)
 
+import warnings
+warnings.simplefilter('ignore')
```

Comment:
Why?

Reply:
I have a weird behaviour in this branch that I haven't figure out why it happens and how to solve it decently. The problem is the following: When I run the tests (or any other python program) with python inside the virtualenv created with this branch, the interpreter prints a lot of warnings (most of them are ResourceWarning) and it floods the output of the test cases. The weird thing is that those warnings are also present in angr-master (if printed explicitly) but they are in some way hidden.

I tried to investigate changes in the environment between angr-master and angr-feat/mixed_java but the environment and the python packages installed are exactly the same. Launching the script with `python -W ignore` or setting the evironment variable `PYTHONWARNINGS="ignore"` does not solve the problem.

The steps to reproduce the behavior are the following:

- `(angr-feat/mixed_java) python tests/test_0div.py ---> LOT OF WARNINGS PRINTED` 

- `(angr-master) python tests/test_0div.py ---> NO WARNING PRINTED` 

- `(angr-master) python -W always tests/test_0div.py ---> LOT OF WARNINGS PRINTED (same warnings printed in the branch feat/mixed_java)`

I put those two lines as a temporary fix to not flood your output during the execution of the test cases... I should have put a comment :sweat_smile:.

Have you ever encountered something similar?

Reply:
No. It's possible that one of the other updated libraries are forcing all logging output to be printed out.

Reply:
Now it's gone!